src/scheduler/App.js

```diff
@@ -13,10 +13,7 @@ export const App = () => {
 
   options = { init: setIntialState, ...window.schedulerOptions };
 
-  const providerState = options.init({
-    dayjs,
-    defaultState: defaultState()
```

Comment:
This arg doesn't seem to be doing anything, at least for my use case

Reply:
The idea here was to provide a default state so the end user only needs to override "custom" options.  This will vary but I would like to keep this i.e for cases where you only want to change one thing.
```
{...defaultState, yourState}
```

Also if the end user forgets to set a "required" piece of state that could cause issues.